mobile-web/src/amp-url-creator.js

```diff
@@ -64,23 +83,46 @@ const MAX_DOMAIN_LABEL_LENGTH_ = 63;
  */
 export function constructViewerCacheUrl(url, initParams,
   opt_cacheUrlAuthority, opt_viewerJsVersion) {
+	  return constructViewerCacheUrlOptions(url, false, initParams, opt_cacheUrlAuthority, opt_viewerJsVersion);
+ }
+
+/**
+ * Constructs a Viewer cache url using these rules:
+ * https://developers.google.com/amp/cache/overview
+ * 
+ * Example:
+ * Input url 'http://ampproject.org' can return 
+ * 'https://www-ampproject-org.cdn.ampproject.org/v/s/www.ampproject.org/?amp_js_v=0.1#origin=http%3A%2F%2Flocalhost%3A8000'
+ * 
+ * @param {string} url The complete publisher url.
+ * @param {object} initParams Params containing origin, etc.
+ * @param (boolean} native Whether or not the url generated follows rules for native viewers (like AMPKit)
+ * @param {string} opt_cacheUrlAuthority
+ * @param {string} opt_viewerJsVersion
+ * @return {!Promise<string>}
+ * @private
+ */
+function constructViewerCacheUrlOptions(url, native, initParams,
```

Comment:
change to isNative to show it's a boolean

Reply:
Done.